commit 112e3457b2d37aeb9c27d320698c8a960ac0a997
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Jul 7 04:24:14 2021 -0300

    obl_nmod .*nte e .*vel
    
    NOUN filho de NOUN, caso haja amod .*nte ou .*vel, provavelmente deverá ser obl desse adjetivo
    
    partial obl_nmod elvis
    
    squash

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index dbf4c42..4795448 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1415,7 +1415,7 @@
 4-5	no	_	_	_	_	_	_	_	_
 4	em	em	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	2	nmod	B=INSTRUMENTO	_
+6	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	3	obl	B=INSTRUMENTO	_
 7	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	8	aux:pass	O	_
 8	interpretadas	interpretar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 9	com	com	ADP	_	_	11	case	O	_
@@ -4681,7 +4681,7 @@
 15	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	6	obl	O	_
 16	predominantes	predominante	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	de	de	ADP	_	_	18	case	O	_
-18	águas	água	NOUN	_	Gender=Fem|Number=Plur	15	nmod	B=FLUIDO	_
+18	águas	água	NOUN	_	Gender=Fem|Number=Plur	16	obl	B=FLUIDO	_
 19	rasas	raso	ADJ	_	Gender=Fem|Number=Plur	18	amod	O	_
 20	(	(	PUNCT	_	_	21	punct	O	_
 21	Milani	Milani	PROPN	_	Gender=Masc|Number=Sing	4	parataxis	O	_
@@ -5991,7 +5991,7 @@
 32-33	na	_	_	_	_	_	_	_	_
 32	em	em	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
-34	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	29	nmod	B=BACIA:TIPO	_
+34	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	31	obl	B=BACIA:TIPO	_
 35	(	(	PUNCT	_	_	36	punct	O	_
 36	Fontana	Fontana	PROPN	_	Gender=Masc|Number=Sing	5	parataxis	O	_
 37	1996	1996	PROPN	_	Number=Sing	36	flat:name	O	_
@@ -7357,7 +7357,7 @@
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Lineamento	Lineamento	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	Lineamento	Lineamento	PROPN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	Tibagi	Tibagi	PROPN	_	Number=Sing	6	flat:name	O	_
 8	e	e	CCONJ	_	_	9	cc	O	_
 9	“	“	PROPN	_	Gender=Masc|Number=Sing	6	conj	O	_
@@ -10160,7 +10160,7 @@
 16-17	na	_	_	_	_	_	_	_	_
 16	em	em	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
-18	região	região	NOUN	_	Gender=Fem|Number=Sing	13	nmod	O	_
+18	região	região	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 19	continental	continental	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 20	,	,	PUNCT	_	_	13	punct	O	_
 21	já	já	ADV	_	_	27	mark	O	MWEPOS=SCONJ
@@ -11500,7 +11500,7 @@
 4-5	deste	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	6	det	O	_
-6	processo	processo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	processo	processo	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	,	,	PUNCT	_	_	8	punct	O	_
 8	então	então	ADV	_	_	10	advmod	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
@@ -12184,10 +12184,10 @@
 26	agente	agente	NOUN	_	Gender=Masc|Number=Sing	22	obl	O	_
 27	emulsificante	emulsificante	ADJ	_	Gender=Masc|Number=Sing	26	amod	B=QUÍMICA	_
 28	,	,	PUNCT	_	_	32	punct	O	_
-29	de	de	ADP	_	_	30	case	O	_
-30	modo	modo	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
-31	a	a	SCONJ	_	_	32	mark	O	_
-32	impedir	impedir	VERB	_	VerbForm=Inf	30	acl	O	_
+29	de	de	ADP	_	_	32	mark	O	MWEPOS=SCONJ
+30	modo	modo	NOUN	_	Gender=Masc|Number=Sing	29	fixed	O	_
+31	a	a	SCONJ	_	_	29	fixed	O	_
+32	impedir	impedir	VERB	_	VerbForm=Inf	13	advcl	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
 34	união	união	NOUN	_	Gender=Fem|Number=Sing	32	obj	O	_
 35-36	dessas	_	_	_	_	_	_	_	_
@@ -13160,7 +13160,7 @@
 17-18	da	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	15	nmod	B=QUÍMICA	_
+19	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	16	obl	B=QUÍMICA	_
 20	.	.	PUNCT	_	_	1	punct	O	_
 21	.	.	PUNCT	_	_	1	punct	O	_
 
@@ -15124,7 +15124,7 @@
 34-35	na	_	_	_	_	_	_	_	_
 34	em	em	ADP	_	_	36	case	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
-36	fase	fase	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
+36	fase	fase	NOUN	_	Gender=Fem|Number=Sing	33	obl	O	_
 37	aquosa	aquoso	ADJ	_	Gender=Fem|Number=Sing	36	amod	O	_
 38	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	40	cop	O	_
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
@@ -16313,7 +16313,7 @@
 22-23	da	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
-24	evaporação	evaporação	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
+24	evaporação	evaporação	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 25-26	da	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
@@ -17863,7 +17863,7 @@
 19-20	no	_	_	_	_	_	_	_	_
 19	em	em	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	valor	valor	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
+21	valor	valor	NOUN	_	Gender=Masc|Number=Sing	18	obl	O	_
 22	final	final	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
@@ -18163,7 +18163,7 @@
 57-58	na	_	_	_	_	_	_	_	_
 57	em	em	ADP	_	_	59	case	O	_
 58	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	59	det	O	_
-59	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	54	nmod	B=QUÍMICA	_
+59	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	56	obl	B=QUÍMICA	_
 60	.	.	PUNCT	_	_	8	punct	O	_
 61	.	.	PUNCT	_	_	8	punct	O	_
 
@@ -19134,7 +19134,7 @@
 57-58	na	_	_	_	_	_	_	_	_
 57	em	em	ADP	_	_	59	case	O	_
 58	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	59	det	O	_
-59	especificação	especificação	NOUN	_	Gender=Fem|Number=Sing	55	nmod	O	_
+59	especificação	especificação	NOUN	_	Gender=Fem|Number=Sing	56	obl	O	_
 60	(	(	PUNCT	_	_	61	punct	O	_
 61	DNER-EM	DNER-EM	PROPN	_	Gender=Fem|Number=Sing	55	appos	O	_
 62	369/97	369/97	NUM	_	NumType=Card	61	nummod	O	_
@@ -21756,7 +21756,7 @@
 80-81	nas	_	_	_	_	_	_	_	_
 80	em	em	ADP	_	_	82	case	O	_
 81	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	82	det	O	_
-82	porções	porção	NOUN	_	Gender=Fem|Number=Plur	78	nmod	O	_
+82	porções	porção	NOUN	_	Gender=Fem|Number=Plur	79	nmod	O	_
 83	mais	mais	ADV	_	_	84	advmod	O	_
 84	distais	distal	ADJ	_	Gender=Fem|Number=Plur	82	amod	O	_
 85	(	(	PUNCT	_	_	86	punct	O	_
@@ -23908,7 +23908,7 @@
 4-5	nesta	_	_	_	_	_	_	_	_
 4	em	em	ADP	_	_	6	case	O	_
 5	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	6	det	O	_
-6	época	época	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+6	época	época	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7	gerou	gerar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 8	siltitos	siltito	NOUN	_	Gender=Masc|Number=Plur	7	obj	B=LITOLOGIA	_
 9	e	e	CCONJ	_	_	10	cc	O	_
@@ -24660,7 +24660,7 @@
 12	faunística	faunístico	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	coerente	coerente	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 14	para	para	ADP	_	_	15	case	O	_
-15	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
+15	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
 16	marinho	marinho	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
 17	raso	raso	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
 18	.	.	PUNCT	_	_	9	punct	O	_
@@ -25605,7 +25605,7 @@
 38-39	da	_	_	_	_	_	_	_	_
 38	de	de	ADP	_	_	40	case	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
-40	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	36	nmod	O	_
+40	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	37	obl	O	_
 41	,	,	PUNCT	_	_	42	punct	O	_
 42	apresentando	apresentar	VERB	_	VerbForm=Ger	34	advcl	O	_
 43	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	44	det	O	_
@@ -25725,7 +25725,7 @@
 30-31	da	_	_	_	_	_	_	_	_
 30	de	de	ADP	_	_	32	case	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
-32	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
+32	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
 33	,	,	PUNCT	_	_	34	punct	O	_
 34-35	tornando-se	_	_	_	_	_	_	_	_
 34	tornando	tornar	VERB	_	VerbForm=Ger	20	advcl	O	_
@@ -25949,7 +25949,7 @@
 4	granodecrescência	granodecrescência	NOUN	_	Gender=Fem|Number=Sing	3	obj	O	_
 5	ascendente	ascendente	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	para	para	ADP	_	_	7	case	O	_
-7	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	4	nmod	B=LITOLOGIA	_
+7	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	3	obl	B=LITOLOGIA	_
 8	médios	médio	ADJ	_	Gender=Masc|Number=Plur	7	amod	O	_
 9	a	a	ADP	_	_	10	case	O	_
 10	grossos	grosso	ADJ	_	Gender=Masc|Number=Plur	7	amod	O	_
@@ -29643,7 +29643,7 @@
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	3	nmod	B=LITOLOGIA	_
+7	fácies	fácies	NOUN	_	Gender=Fem|Number=Sing	3	nmod	B=LITOLOGIA	_
 8	coquinas	coquina	ADJ	_	Gender=Fem|Number=Sing	10	amod	B=LITOLOGIA	_
 9	(	(	PUNCT	_	_	10	punct	O	_
 10	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	7	appos	B=LITOLOGIA	_
@@ -31395,7 +31395,7 @@
 8-9	ao	_	_	_	_	_	_	_	_
 8	a	a	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	Membro	membro	PROPN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+10	Membro	membro	PROPN	_	Gender=Masc|Number=Sing	7	obl	O	_
 11	Romualdo	Romualdo	PROPN	_	Number=Sing	10	flat:name	O	_
 12	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	aux:pass	O	_
 13	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -32005,7 +32005,7 @@
 42	feição	feição	NOUN	_	Gender=Fem|Number=Sing	38	xcomp	O	_
 43	resultante	resultante	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 44	de	de	ADP	_	_	45	case	O	_
-45	exposições	exposição	NOUN	_	Gender=Fem|Number=Plur	42	nmod	O	_
+45	exposições	exposição	NOUN	_	Gender=Fem|Number=Plur	43	obl	O	_
 46	subaéreas	subaéreo	ADJ	_	Gender=Fem|Number=Plur	45	amod	O	_
 47	com	com	ADP	_	_	48	case	O	_
 48	retrabalhamento	retrabalhamento	NOUN	_	Gender=Masc|Number=Sing	45	nmod	B=LOCAL:GEO	_
@@ -32130,7 +32130,7 @@
 23	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
 24	para	para	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	base	base	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
+26	base	base	NOUN	_	Gender=Fem|Number=Sing	19	obl	O	_
 27-28	do	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -34359,7 +34359,7 @@
 6	importantes	importante	ADJ	_	Gender=Fem|Number=Plur	5	amod	O	_
 7	em	em	ADP	_	_	9	case	O	_
 8	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	produção	produção	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -34576,7 +34576,7 @@
 6	constituintes	constituinte	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	de	de	ADP	_	_	9	case	O	_
 8	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -36064,7 +36064,7 @@
 17	:	:	PUNCT	_	_	20	punct	O	_
 18			PUNCT	_	_	20	punct	O	_
 19	Por	por	ADP	_	_	20	case	O	_
-20	condução	condução	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
+20	condução	condução	NOUN	_	Gender=Fem|Number=Sing	15	appos	O	_
 21	:	:	PUNCT	_	_	23	punct	O	_
 22	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	aux:pass	O	_
 23	baseado	basear	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	20	acl:relcl	O	_
@@ -37625,7 +37625,7 @@
 30	éter	éter	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 31	existente	existente	ADJ	_	Gender=Masc|Number=Sing	30	amod	O	_
 32	entre	entre	ADP	_	_	33	case	O	_
-33	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	30	nmod	O	_
+33	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	31	obl	O	_
 34	.	.	PUNCT	_	_	15	punct	O	_
 35	.	.	PUNCT	_	_	15	punct	O	_
 
@@ -38784,7 +38784,7 @@
 6-7	no	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
+8	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
 9	,	,	PUNCT	_	_	4	punct	O	_
 10	então	então	ADV	_	_	4	advmod	O	_
 11	e	e	CCONJ	_	_	16	cc	O	_
@@ -39856,7 +39856,7 @@
 41-42	no	_	_	_	_	_	_	_	_
 41	em	em	ADP	_	_	43	case	O	_
 42	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	43	det	O	_
-43	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	38	nmod	O	_
+43	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	40	obl	O	_
 44	,	,	PUNCT	_	_	36	punct	O	_
 45	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	46	nsubj	O	_
 46	aquece	aquecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	38	acl:relcl	O	_
@@ -40704,7 +40704,7 @@
 17-18	nos	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
-19	reservatórios	reservatório	NOUN	_	Gender=Masc|Number=Plur	15	nmod	O	_
+19	reservatórios	reservatório	NOUN	_	Gender=Masc|Number=Plur	16	obl	O	_
 20	considerados	considerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	19	acl	O	_
 21	para	para	ADP	_	_	23	case	O	_
 22	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	23	det	O	_
@@ -40868,7 +40868,7 @@
 3	importante	importante	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	para	para	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	indutivo	indutivo	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	cop	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
@@ -41644,11 +41644,11 @@
 35	de	de	ADP	_	_	37	case	O	_
 36	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	37	det	O	_
 37	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	34	nmod	O	_
-38	decorrente	decorrente	ADJ	_	Gender=Fem|Number=Sing	37	amod	O	_
+38	decorrente	decorrente	ADJ	_	Gender=Fem|Number=Sing	34	amod	O	_
 39-40	deste	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	41	det	O	_
-41	acréscimo	acréscimo	NOUN	_	Gender=Masc|Number=Sing	37	nmod	O	_
+41	acréscimo	acréscimo	NOUN	_	Gender=Masc|Number=Sing	38	obl	O	_
 42-43	da	_	_	_	_	_	_	_	_
 42	de	de	ADP	_	_	44	case	O	_
 43	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	44	det	O	_
@@ -41663,7 +41663,7 @@
 3	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	acl:relcl	O	_
 4	ser	ser	AUX	_	VerbForm=Inf	6	cop	O	_
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
+6	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	3	xcomp	O	_
 7	paramétrico	paramétrico	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	viável	viável	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 9	de	de	ADP	_	_	10	case	O	_
@@ -41671,7 +41671,7 @@
 11	de	de	ADP	_	_	12	case	O	_
 12	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	10	nmod	B=FLUIDO	_
 13	por	por	ADP	_	_	14	case	O	_
-14	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+14	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 15	eletromagnético	eletromagnético	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	.	.	PUNCT	_	_	1	punct	O	_
 17	.	.	PUNCT	_	_	1	punct	O	_
@@ -42674,7 +42674,7 @@
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	água	água	NOUN	_	Gender=Fem|Number=Sing	18	nmod	B=FLUIDO	_
+23	água	água	NOUN	_	Gender=Fem|Number=Sing	20	obl	B=FLUIDO	_
 24	.	.	PUNCT	_	_	4	punct	O	_
 25	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -42951,7 +42951,7 @@
 6	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	1	ccomp	O	_
 7	interessante	interessante	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	para	para	ADP	_	_	9	case	O	_
-9	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
+9	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=FLUIDO	_
 12	.	.	PUNCT	_	_	1	punct	O	_
@@ -48268,7 +48268,7 @@
 15	cascalho	cascalho	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 16	provenientes	proveniente	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	de	de	ADP	_	_	18	case	O	_
-18	poços	poço	NOUN	_	Gender=Masc|Number=Plur	15	nmod	O	_
+18	poços	poço	NOUN	_	Gender=Masc|Number=Plur	16	obl	O	_
 19	perfurados	perfurar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	13	acl	O	_
 20	com	com	ADP	_	_	21	case	O	_
 21	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	19	obl	O	_
@@ -52036,7 +52036,7 @@
 39-40	das	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	41	det	O	_
-41	folhas	folha	NOUN	_	Gender=Fem|Number=Plur	37	nmod	O	_
+41	folhas	folha	NOUN	_	Gender=Fem|Number=Plur	38	obl	O	_
 42	de	de	ADP	_	_	43	case	O	_
 43	silicato	silicato	NOUN	_	Gender=Masc|Number=Sing	41	nmod	B=LITOLOGIA	_
 44	e	e	CCONJ	_	_	46	cc	O	_
@@ -52090,7 +52090,7 @@
 24-25	da	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	água	água	NOUN	_	Gender=Fem|Number=Sing	22	nmod	B=FLUIDO	_
+26	água	água	NOUN	_	Gender=Fem|Number=Sing	23	obl	B=FLUIDO	_
 27	líquida	líquido	ADJ	_	Gender=Fem|Number=Sing	26	amod	I=FLUIDO	_
 28	.	.	PUNCT	_	_	2	punct	O	_
 29	.	.	PUNCT	_	_	2	punct	O	_
@@ -56312,7 +56312,7 @@
 10-11	do	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	aço	aço	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
+12	aço	aço	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 13	inox	inox	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	,	,	PUNCT	_	_	16	punct	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -58673,7 +58673,7 @@
 10-11	do	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	aço	aço	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
+12	aço	aço	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 13	inox	inox	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	,	,	PUNCT	_	_	16	punct	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -72190,7 +72190,7 @@
 5-6	nesta	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	7	det	O	_
-7	localidade	localidade	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+7	localidade	localidade	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8	também	também	ADV	_	_	9	advmod	O	_
 9	corrobora	corroborar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -75659,7 +75659,7 @@
 5	paleoventos	paleovento	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 6	predominantes	predominante	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	para	para	ADP	_	_	8	case	O	_
-8	SSW	SSW	PROPN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+8	SSW	SSW	PROPN	_	Gender=Masc|Number=Sing	6	obl	O	_
 9	,	,	PUNCT	_	_	11	punct	O	_
 10	com	com	ADP	_	_	11	case	O	_
 11	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
@@ -76226,7 +76226,7 @@
 5-6	no	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-7	Eocretáceo	Eocretáceo	PROPN	_	Gender=Masc|Number=Sing	3	nmod	B=GEOCRONOLOGIA	_
+7	Eocretáceo	Eocretáceo	PROPN	_	Gender=Masc|Number=Sing	4	obl	B=GEOCRONOLOGIA	_
 8	tinham	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	0	root	O	_
 9	direção	direção	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
 10	NW-SE	NW-SE	PROPN	_	Gender=Fem|Number=Sing	9	nmod	O	_
@@ -77880,7 +77880,7 @@
 233-234	no	_	_	_	_	_	_	_	_
 233	em	em	ADP	_	_	235	case	O	_
 234	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	235	det	O	_
-235	Eocretáceo	Eocretáceo	PROPN	_	Gender=Masc|Number=Sing	230	nmod	B=GEOCRONOLOGIA	_
+235	Eocretáceo	Eocretáceo	PROPN	_	Gender=Masc|Number=Sing	232	obl	B=GEOCRONOLOGIA	_
 236-237	da	_	_	_	_	_	_	_	_
 236	de	de	ADP	_	_	235	flat:name	O	_
 237	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	235	flat:name	O	_
@@ -77970,7 +77970,7 @@
 318-319	no	_	_	_	_	_	_	_	_
 318	em	em	ADP	_	_	320	case	O	_
 319	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	320	det	O	_
-320	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	316	nmod	B=GEOCRONOLOGIA	_
+320	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	317	obl	B=GEOCRONOLOGIA	_
 321	Inferior	Inferior	PROPN	_	Number=Sing	320	flat:name	I=GEOCRONOLOGIA	_
 322	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	297	advcl	O	_
 323	condicionamento	condicionamento	NOUN	_	Gender=Masc|Number=Sing	322	obj	O	_
@@ -78608,7 +78608,7 @@
 62-63	do	_	_	_	_	_	_	_	_
 62	de	de	ADP	_	_	64	case	O	_
 63	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	64	det	O	_
-64	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	60	nmod	O	_
+64	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	61	obl	O	_
 65	.	.	PUNCT	_	_	31	punct	O	_
 66	.	.	PUNCT	_	_	31	punct	O	_
 
@@ -78830,7 +78830,7 @@
 61-62	da	_	_	_	_	_	_	_	_
 61	de	de	ADP	_	_	63	case	O	_
 62	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	63	det	O	_
-63	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	59	nmod	O	_
+63	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	60	obl	O	_
 64	avançada	avançar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	63	acl	O	_
 65	de	de	ADP	_	_	66	case	O	_
 66	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	63	nmod	B=FLUIDO	_
@@ -80171,7 +80171,7 @@
 2	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	16	nsubj	O	_
 3	resultante	resultante	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	de	de	ADP	_	_	5	case	O	_
-5	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+5	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 6	e	e	CCONJ	_	_	7	cc	O	_
 7	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	5	conj	B=QUÍMICA	_
 8	(	(	PUNCT	_	_	9	punct	O	_
@@ -83296,7 +83296,7 @@
 42-43	no	_	_	_	_	_	_	_	_
 42	em	em	ADP	_	_	44	case	O	_
 43	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	44	det	O	_
-44	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	40	nmod	O	_
+44	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	41	obl	O	_
 45	,	,	PUNCT	_	_	46	punct	O	_
 46	facilitando	facilitar	VERB	_	VerbForm=Ger	29	advcl	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
@@ -84120,7 +84120,7 @@
 17-18	no	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
-19	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
+19	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
 20	,	,	PUNCT	_	_	4	punct	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	23	nsubj	B=QUÍMICA	_
@@ -84319,7 +84319,7 @@
 34-35	no	_	_	_	_	_	_	_	_
 34	em	em	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
-36	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
+36	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	33	obl	O	_
 37	.	.	PUNCT	_	_	22	punct	O	_
 38	.	.	PUNCT	_	_	22	punct	O	_
 
@@ -90318,7 +90318,7 @@
 6	relevantes	relevante	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	para	para	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	,	,	PUNCT	_	_	2	punct	O	_
 11	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	16	obl	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -97515,7 +97515,7 @@
 15	importantes	importante	ADJ	_	Gender=Masc|Number=Plur	14	amod	O	_
 16	para	para	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
+18	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 19	.	.	PUNCT	_	_	11	punct	O	_
 20	.	.	PUNCT	_	_	11	punct	O	_
 
@@ -98773,18 +98773,18 @@
 14	que	que	SCONJ	_	_	13	fixed	O	_
 15	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	cop	O	_
 16	inviável	inviável	ADJ	_	Gender=Masc|Number=Sing	3	advcl	O	_
-17	que	que	SCONJ	_	_	20	mark	O	_
+17	que	que	SCONJ	_	_	25	mark	O	_
 18	todo	todo	DET	_	Gender=Masc|Number=Sing|PronType=Tot	20	det	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
-20	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	16	nsubj	B=FLUIDO	_
+20	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	25	nsubj	B=FLUIDO	_
 21	presente	presente	ADJ	_	Gender=Masc|Number=Sing	20	amod	O	_
 22-23	no	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
-25	entre	entre	ADP	_	_	27	case	O	_
+24	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
+25	entre	entre	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	ccomp	O	_
 26	em	em	ADP	_	_	27	case	O	_
-27	contato	contato	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
+27	contato	contato	NOUN	_	Gender=Masc|Number=Sing	25	obj	O	_
 28	com	com	ADP	_	_	29	case	O	_
 29	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	27	nmod	B=QUÍMICA	_
 30	injetado	injetar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	29	acl	O	_
@@ -104230,7 +104230,7 @@
 10-11	da	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	combustão	combustão	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
+12	combustão	combustão	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	de	de	ADP	_	_	16	case	O	_
 14	todo	todo	DET	_	Gender=Masc|Number=Sing|PronType=Tot	16	det	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -104324,7 +104324,7 @@
 17-18	do	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
-19	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
+19	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	captura	captura	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
@@ -104667,7 +104667,7 @@
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	consistem	consistir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	basicamente	basicamente	ADV	_	_	7	advmod	O	_
 9-10	nas	_	_	_	_	_	_	_	_
@@ -104678,7 +104678,7 @@
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
+15	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 16	energético	energético	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
 17	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	19	nsubj:pass	O	_
 18	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	19	aux:pass	O	_
@@ -104800,7 +104800,7 @@
 65-66	do	_	_	_	_	_	_	_	_
 65	de	de	ADP	_	_	67	case	O	_
 66	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	67	det	O	_
-67	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	63	nmod	O	_
+67	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	64	obl	O	_
 68	energético	energético	ADJ	_	Gender=Masc|Number=Sing	67	amod	O	_
 69	adicional	adicional	ADJ	_	Gender=Masc|Number=Sing	67	amod	O	_
 70-71	na	_	_	_	_	_	_	_	_
@@ -105118,7 +105118,7 @@
 87-88	do	_	_	_	_	_	_	_	_
 87	de	de	ADP	_	_	89	case	O	_
 88	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	89	det	O	_
-89	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	85	nmod	O	_
+89	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	86	obl	O	_
 90	energético	energético	ADJ	_	Gender=Masc|Number=Sing	89	amod	O	_
 91-92	da	_	_	_	_	_	_	_	_
 91	de	de	ADP	_	_	93	case	O	_
@@ -105191,7 +105191,7 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	10	nsubj	O	_
 3	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
-4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	transporte	transporte	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 7	de	de	ADP	_	_	8	case	O	_
@@ -105374,7 +105374,7 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	10	nsubj	O	_
 3	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
-4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	armazenamento	armazenamento	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 7	de	de	ADP	_	_	8	case	O	_
@@ -105643,7 +105643,7 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	11	nsubj	O	_
 3	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
-4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+4	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	reciclagem	reciclagem	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 7-8	do	_	_	_	_	_	_	_	_
@@ -105805,7 +105805,7 @@
 
 # text = Utilizando todos os valores calculados anteriormente, a emissão total anual decorrente do gasto energético do projeto será: Emissões decorrentes do — 746.445 + 5.172 + 200.962 + 316.527 = 1.269.106 tCO, Portanto as emissões de projeto serão de: 7.1.3. Emissões fugitivas
 # sent_id = 161-20150810-MONOGRAFIA_0-844
-1	Utilizando	utilizar	VERB	_	VerbForm=Ger	22	advcl	O	_
+1	Utilizando	utilizar	VERB	_	VerbForm=Ger	22	advcl	O	eliminar
 2	todos	todo	DET	_	Gender=Masc|Number=Plur|PronType=Tot	4	det	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
 4	valores	valor	NOUN	_	Gender=Masc|Number=Plur	1	obj	O	_
@@ -105820,7 +105820,7 @@
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
+15	gasto	gasto	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 16	energético	energético	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
 17-18	do	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
@@ -106483,7 +106483,7 @@
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
+28	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
 29	extra	extra	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	28	nmod	B=FLUIDO	_
@@ -108753,7 +108753,7 @@
 23-24	da	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	costa	costa	NOUN	_	Gender=Fem|Number=Sing	21	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+25	costa	costa	NOUN	_	Gender=Fem|Number=Sing	22	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	,	,	PUNCT	_	_	21	punct	O	_
 27	também	também	ADV	_	_	28	advmod	O	_
 28	resultam	resultar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -110444,7 +110444,7 @@
 99-100	das	_	_	_	_	_	_	_	_
 99	de	de	ADP	_	_	101	case	O	_
 100	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	101	det	O	_
-101	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	96	nmod	O	_
+101	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	98	obl	O	_
 102	de	de	ADP	_	_	103	case	O	_
 103	sal	sal	NOUN	_	Gender=Masc|Number=Sing	101	nmod	B=LITOLOGIA	_
 104	e	e	CCONJ	_	_	106	cc	O	_
@@ -112998,7 +112998,7 @@
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	poço	poço	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+10	poço	poço	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 11	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	aux:pass	O	_
 12	enviada	enviar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 13	a	a	ADP	_	_	15	case	O	_
@@ -113626,7 +113626,7 @@
 30	matéria	matéria	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	orgânica	orgânico	ADJ	_	Gender=Fem|Number=Sing	30	amod	O	_
 32	por	por	ADP	_	_	33	case	O	_
-33	bactérias	bactéria	NOUN	_	Gender=Fem|Number=Plur	25	obl	O	_
+33	bactérias	bactéria	NOUN	_	Gender=Fem|Number=Plur	28	nmod	O	_
 34	anaeróbicas	anaeróbico	ADJ	_	Gender=Fem|Number=Plur	33	amod	O	_
 35	e	e	CCONJ	_	_	38	cc	O	_
 36-37	pela	_	_	_	_	_	_	_	_
@@ -114226,7 +114226,7 @@
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	pré-sal	pré-sal	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
+9	pré-sal	pré-sal	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	cop	O	_
 11	em	em	ADP	_	_	13	case	O	_
 12	constante	constante	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
@@ -116354,7 +116354,7 @@
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	aux:pass	O	_
 11	composta	compor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 12	basicamente	basicamente	ADV	_	_	11	advmod	O	_
@@ -116795,7 +116795,7 @@
 10-11	do	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	separador	separador	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
+12	separador	separador	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 13	gravitacional	gravitacional	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	,	,	PUNCT	_	_	14	punct	O	_
@@ -119028,7 +119028,7 @@
 32-33	dos	_	_	_	_	_	_	_	_
 32	de	de	ADP	_	_	34	case	O	_
 33	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	34	det	O	_
-34	poços	poço	NOUN	_	Gender=Masc|Number=Plur	30	nmod	O	_
+34	poços	poço	NOUN	_	Gender=Masc|Number=Plur	31	obl	O	_
 35	:	:	PUNCT	_	_	36	punct	O	_
 36	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	30	appos	B=FLUIDO	_
 37	,	,	PUNCT	_	_	38	punct	O	_
@@ -119677,7 +119677,7 @@
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
-30	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
+30	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
 31	a	a	ADP	_	_	33	case	O	_
 32	grandes	grande	ADJ	_	Gender=Fem|Number=Plur	33	amod	O	_
 33	profundidades	profundidade	NOUN	_	Gender=Fem|Number=Plur	30	nmod	O	_
@@ -119812,9 +119812,9 @@
 61-62	do	_	_	_	_	_	_	_	_
 61	de	de	ADP	_	_	63	case	O	_
 62	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	63	det	O	_
-63	poço	poço	NOUN	_	Gender=Masc|Number=Sing	59	nmod	O	_
+63	poço	poço	NOUN	_	Gender=Masc|Number=Sing	60	obl	O	_
 64	em	em	ADP	_	_	65	case	O	_
-65	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	59	nmod	O	_
+65	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	53	nmod	O	_
 66	submarino	submarino	ADJ	_	Gender=Masc|Number=Sing	65	amod	B=TRANSPORTE	_
 67	,	,	PUNCT	_	_	68	punct	O	_
 68	fases	fase	NOUN	_	Gender=Fem|Number=Plur	59	appos	O	_
@@ -119830,7 +119830,7 @@
 78	estacionárias	estacionário	ADJ	_	Gender=Fem|Number=Plur	77	amod	O	_
 79	de	de	ADP	_	_	80	case	O	_
 80	produção	produção	NOUN	_	Gender=Fem|Number=Sing	77	nmod	O	_
-81	.	.	PUNCT	_	_	83	punct	O	_
+81	.	.	PUNCT	_	_	83	punct	O	sentenciação
 82	•	•	PUNCT	_	_	83	punct	O	_
 83	Cenário	cenário	NOUN	_	Gender=Masc|Number=Sing	77	appos	O	_
 84	subsea	subsea	X	_	_	83	flat:foreign	O	_
@@ -122565,7 +122565,7 @@
 29	plataformas	plataforma	NOUN	_	Gender=Fem|Number=Plur	26	appos	O	_
 30	existentes	existente	ADJ	_	Gender=Fem|Number=Plur	29	amod	O	_
 31	em	em	ADP	_	_	32	case	O	_
-32	águas	água	NOUN	_	Gender=Fem|Number=Plur	29	nmod	B=FLUIDO	_
+32	águas	água	NOUN	_	Gender=Fem|Number=Plur	30	obl	B=FLUIDO	_
 33	rasas	raso	ADJ	_	Gender=Fem|Number=Plur	32	amod	O	_
 34	ou	ou	CCONJ	_	_	35	cc	O	_
 35	onshore	onshore	NOUN	_	Gender=Masc|Number=Sing	33	conj	O	_
@@ -123449,10 +123449,10 @@
 
 # text = Dentre os principais desafios no desenvolvimento desta tecnologia estão presentes garantia de escoamento, proteção contra corrosão, injeção química, escoamento multifásico e formação de emulsão (EUPHEMIO, 2009)..
 # sent_id = 334-20150521-MONOGRAFIA_0-377
-1	Dentre	dentre	ADV	_	_	13	advmod	O	_
+1	Dentre	dentre	ADP	_	_	4	case	O	_
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
 3	principais	principal	ADJ	_	Gender=Masc|Number=Plur	4	amod	O	_
-4	desafios	desafio	NOUN	_	Gender=Masc|Number=Plur	13	nsubj	O	_
+4	desafios	desafio	NOUN	_	Gender=Masc|Number=Plur	12	nsubj	O	_
 5-6	no	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -123461,9 +123461,9 @@
 8	de	de	ADP	_	_	10	case	O	_
 9	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	10	det	O	_
 10	tecnologia	tecnologia	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
-11	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	cop	O	_
-12	presentes	presente	ADJ	_	Gender=Fem|Number=Plur	13	amod	O	_
-13	garantia	garantia	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
+11	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	cop	O	_
+12	presentes	presente	ADJ	_	Gender=Fem|Number=Plur	0	root	O	_
+13	garantia	garantia	NOUN	_	Gender=Fem|Number=Sing	12	xcomp	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	escoamento	escoamento	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
@@ -123481,12 +123481,12 @@
 28	de	de	ADP	_	_	29	case	O	_
 29	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	27	nmod	B=QUÍMICA	_
 30	(	(	PUNCT	_	_	31	punct	O	_
-31	EUPHEMIO	EUPHEMIO	PROPN	_	Gender=Masc|Number=Sing	13	parataxis	O	_
+31	EUPHEMIO	EUPHEMIO	PROPN	_	Gender=Masc|Number=Sing	12	parataxis	O	_
 32	,	,	PUNCT	_	_	31	flat:name	O	_
 33	2009	2009	NUM	_	NumType=Card	31	flat:name	O	_
 34	)	)	PUNCT	_	_	31	punct	O	_
-35	.	.	PUNCT	_	_	13	punct	O	_
-36	.	.	PUNCT	_	_	13	punct	O	_
+35	.	.	PUNCT	_	_	12	punct	O	_
+36	.	.	PUNCT	_	_	12	punct	O	_
 
 # text = Um dos principais exemplos de desenvolvimento e aplicação dos dutos submarinos é o campo de Snøhvit localizado no Mar de Barents, a 140 quilômetros da costa da Noruega..
 # sent_id = 334-20150521-MONOGRAFIA_0-378
@@ -123651,7 +123651,7 @@
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	não	não	ADV	_	Polarity=Neg	11	advmod	O	_
 11	processada	processar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	9	acl	O	_
 12	representa	representar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -126341,7 +126341,7 @@
 9-10	dos	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
-11	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	7	nmod	O	_
+11	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
 12	termodinâmico	termodinâmico	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 13	e	e	CCONJ	_	_	14	cc	O	_
 14	cinético	cinético	ADJ	_	Gender=Masc|Number=Sing	12	conj	O	_
@@ -126766,7 +126766,7 @@
 11-12	no	_	_	_	_	_	_	_	_
 11	em	em	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	duto	duto	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=LOCAL:GEO	_
+13	duto	duto	NOUN	_	Gender=Masc|Number=Sing	10	obl	B=LOCAL:GEO	_
 14	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	aux:pass	O	_
 15	constituída	constituir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 16	basicamente	basicamente	ADV	_	_	15	advmod	O	_
@@ -128110,7 +128110,7 @@
 14-15	no	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	gás	gás	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=FLUIDO	_
+16	gás	gás	NOUN	_	Gender=Masc|Number=Sing	13	obl	B=FLUIDO	_
 17	..	..	PUNCT	_	_	4	punct	O	_
 
 # text = Devido a estas diferenças, algumas moléculas permeiam mais rapidamente que outras, outras ainda nem permeiam, permitindo a separação..
@@ -128627,7 +128627,7 @@
 36-37	na	_	_	_	_	_	_	_	_
 36	em	em	ADP	_	_	38	case	O	_
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	O	_
-38	alimentação	alimentação	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
+38	alimentação	alimentação	NOUN	_	Gender=Fem|Number=Sing	35	obl	O	_
 39	,	,	PUNCT	_	_	42	punct	O	_
 40	assim	assim	ADV	_	_	42	cc	O	_
 41	como	como	ADV	_	_	40	fixed	O	_
@@ -130238,7 +130238,7 @@
 20-21	na	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	22	case	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
-22	literatura	literatura	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
+22	literatura	literatura	NOUN	_	Gender=Fem|Number=Sing	19	obl	O	_
 23	(	(	PUNCT	_	_	24	punct	O	_
 24	Colunas	Colunas	PROPN	_	Gender=Fem|Number=Sing	16	parataxis	O	_
 25	1	1	PROPN	_	Number=Sing	24	flat:name	O	_
@@ -131290,7 +131290,7 @@
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	poço	poço	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	sofre	sofrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	10	obj	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -131366,7 +131366,7 @@
 52-53	da	_	_	_	_	_	_	_	_
 52	de	de	ADP	_	_	54	case	O	_
 53	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	54	det	O	_
-54	separação	separação	NOUN	_	Gender=Fem|Number=Sing	50	nmod	O	_
+54	separação	separação	NOUN	_	Gender=Fem|Number=Sing	51	obl	O	_
 55	subsea	subseo	ADJ	_	Gender=Fem|Number=Sing	54	amod	O	_
 56	.	.	PUNCT	_	_	10	punct	O	_
 57	.	.	PUNCT	_	_	10	punct	O	_
@@ -131746,7 +131746,7 @@
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	poço	poço	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	poço	poço	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	,	,	PUNCT	_	_	10	punct	O	_
 8	a	a	ADP	_	_	10	case	O	_
 9	220	220	NUM	_	NumType=Card	10	nummod	O	_
@@ -132086,7 +132086,7 @@
 7-8	deste	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	9	det	O	_
-9	separador	separador	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+9	separador	separador	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	,	,	PUNCT	_	_	3	punct	O	_
 11	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	13	nsubj	O	_
 12	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	13	expl	O	_
@@ -132452,7 +132452,7 @@
 19	de	de	ADP	_	_	22	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 21	primeiro	primeiro	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	22	amod	O	_
-22	separador	separador	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
+22	separador	separador	NOUN	_	Gender=Masc|Number=Sing	18	obl	O	_
 23	,	,	PUNCT	_	_	26	punct	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 25	alta	alto	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
@@ -132545,7 +132545,7 @@
 46-47	da	_	_	_	_	_	_	_	_
 46	de	de	ADP	_	_	48	case	O	_
 47	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	48	det	O	_
-48	separação	separação	NOUN	_	Gender=Fem|Number=Sing	44	nmod	O	_
+48	separação	separação	NOUN	_	Gender=Fem|Number=Sing	45	obl	O	_
 49	subsea	subseo	ADJ	_	Gender=Fem|Number=Sing	48	amod	O	_
 50	.	.	PUNCT	_	_	9	punct	O	_
 51	.	.	PUNCT	_	_	9	punct	O	_
@@ -133943,7 +133943,7 @@
 20-21	no	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	gás	gás	NOUN	_	Gender=Masc|Number=Sing	18	nmod	B=FLUIDO	_
+22	gás	gás	NOUN	_	Gender=Masc|Number=Sing	19	obl	B=FLUIDO	_
 23	,	,	PUNCT	_	_	27	punct	O	_
 24	antes	antes	ADV	_	_	27	mark	O	_
 25-26	deste	_	_	_	_	_	_	_	_
@@ -134974,13 +134974,13 @@
 71	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	67	appos	O	_
 72	rica	rico	ADJ	_	Gender=Fem|Number=Sing	71	amod	O	_
 73	em	em	ADP	_	_	74	case	O	_
-74	água	água	NOUN	_	Gender=Fem|Number=Sing	71	nmod	B=FLUIDO	_
+74	água	água	NOUN	_	Gender=Fem|Number=Sing	72	obl	B=FLUIDO	_
 75	proveniente	proveniente	ADJ	_	Gender=Fem|Number=Sing	74	amod	O	_
 76-77	do	_	_	_	_	_	_	_	_
 76	de	de	ADP	_	_	79	case	O	_
 77	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	79	det	O	_
 78	segundo	segundo	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	79	amod	O	_
-79	separador	separador	NOUN	_	Gender=Masc|Number=Sing	74	nmod	O	_
+79	separador	separador	NOUN	_	Gender=Masc|Number=Sing	75	obl	O	_
 80	subsea	subsea	NOUN	_	Gender=Fem|Number=Sing	79	nmod	O	_
 81	;	;	PUNCT	_	_	83	punct	O	_
 82	•	•	PUNCT	_	_	83	punct	O	_
@@ -135999,33 +135999,33 @@
 3	em	em	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
 5	simulação	simulação	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
-6	diâmetro	diâmetro	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+6	diâmetro	diâmetro	NOUN	_	Gender=Masc|Number=Sing	2	nsubj:pass	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	duto	duto	NOUN	_	Gender=Masc|Number=Sing	6	nmod	B=LOCAL:GEO	_
 9	compatível	compatível	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 10	com	com	ADP	_	_	12	case	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
-12	limitações	limitação	NOUN	_	Gender=Fem|Number=Plur	2	obl	O	_
+12	limitações	limitação	NOUN	_	Gender=Fem|Number=Plur	9	obl	O	_
 13	existentes	existente	ADJ	_	Gender=Fem|Number=Plur	12	amod	O	_
 14-15	no	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	mercado	mercado	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
+16	mercado	mercado	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 17	e	e	CCONJ	_	_	19	cc	O	_
 18	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	19	nsubj	O	_
-19	possibilitam	possibilitar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	2	conj	O	_
+19	possibilitam	possibilitar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	acl:relcl	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	injeção	injeção	NOUN	_	Gender=Fem|Number=Sing	19	obj	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 24	em	em	ADP	_	_	25	case	O	_
-25	quantidade	quantidade	NOUN	_	Gender=Fem|Number=Sing	19	obl	O	_
+25	quantidade	quantidade	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
 26	suficiente	suficiente	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
 27	para	para	SCONJ	_	_	31	mark	O	_
 28	que	que	SCONJ	_	_	27	fixed	O	_
 29	não	não	ADV	_	Polarity=Neg	31	advmod	O	_
 30	sejam	ser	AUX	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	31	aux:pass	O	_
-31	detectadas	detectar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	19	advcl	O	_
+31	detectadas	detectar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	26	ccomp	O	_
 32	formações	formação	NOUN	_	Gender=Fem|Number=Plur	31	nsubj:pass	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	hidratos	hidrato	NOUN	_	Gender=Masc|Number=Plur	32	nmod	O	_
@@ -142162,7 +142162,7 @@
 3	determinante	determinante	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	para	para	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
-6	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+6	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	emulsões	emulsão	NOUN	_	Gender=Fem|Number=Plur	6	nmod	B=QUÍMICA	_
 9	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	cop	O	_
@@ -143597,7 +143597,7 @@
 39-40	ao	_	_	_	_	_	_	_	_
 39	a	a	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
-41	epicentro	epicentro	NOUN	_	Gender=Masc|Number=Sing	37	nmod	O	_
+41	epicentro	epicentro	NOUN	_	Gender=Masc|Number=Sing	38	obl	O	_
 42	,	,	PUNCT	_	_	45	punct	O	_
 43	além	além	ADV	_	_	45	cc	O	_
 44	de	de	ADP	_	_	43	fixed	O	_
@@ -144879,7 +144879,7 @@
 56-57	numa	_	_	_	_	_	_	_	_
 56	em	em	ADP	_	_	58	case	O	_
 57	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	58	det	O	_
-58	crosta	crosta	NOUN	_	Gender=Fem|Number=Sing	54	nmod	O	_
+58	crosta	crosta	NOUN	_	Gender=Fem|Number=Sing	55	obl	O	_
 59	heterogenia	heterogenia	NOUN	_	Gender=Fem|Number=Sing	58	nmod	O	_
 60	devem	dever	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	42	conj	O	_
 61	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	62	expl	O	_
@@ -150588,7 +150588,7 @@
 5	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	4	obj	B=LITOLOGIA	_
 6	resultantes	resultante	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	fluxos	fluxo	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
+8	fluxos	fluxo	NOUN	_	Gender=Masc|Number=Plur	6	obl	O	_
 9	turbidíticos	turbidítico	ADJ	_	Gender=Masc|Number=Plur	8	amod	O	_
 10	densos	denso	ADJ	_	Gender=Masc|Number=Plur	8	amod	O	_
 11	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	12	nsubj	O	_
@@ -153824,7 +153824,7 @@
 24-25	desse	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	26	det	O	_
-26	processo	processo	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
+26	processo	processo	NOUN	_	Gender=Masc|Number=Sing	23	obl	O	_
 27	de	de	ADP	_	_	28	case	O	_
 28	domeamento	domeamento	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
 29	térmico	térmico	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
@@ -155698,7 +155698,7 @@
 11	diferente	diferente	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 12-13	daquelas	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	13	case	O	_
-13	aquelas	aquele	PRON	_	Gender=Fem|Number=Plur|PronType=Dem	10	nmod	O	_
+13	aquelas	aquele	PRON	_	Gender=Fem|Number=Plur|PronType=Dem	11	obl	O	_
 14	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	16	nsubj	O	_
 15	as	o	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	16	nsubj	O	_
 16	circundam	circundar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	acl:relcl	O	_
@@ -156084,7 +156084,7 @@
 7-8	à	_	_	_	_	_	_	_	_
 7	a	a	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-9	pesquisa	pesquisa	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
+9	pesquisa	pesquisa	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 10	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	11	aux:pass	O	_
 11	realizada	realizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -161293,7 +161293,7 @@
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	6	flat:name	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	flat:name	O	_
@@ -161530,7 +161530,7 @@
 
 # text = C a p i t u l o 5 - I n t e r p r e t a ç ã o d o s D a d o s | 78 Na análise dos mapas podem ser observadas cunhas progradantes desde o horizonte Eosantoniano até o horizonte Topo do Cretáceo, representadas no mapa pelo adensamento das isolinhas (Figuras 5.12, 5.13 e 5.14)..
 # sent_id = 258-20160602-TESEMSC_0-455
-1	C	C	PROPN	_	Gender=Masc|Number=Sing	40	nsubj	O	_
+1	C	C	PROPN	_	Gender=Masc|Number=Sing	40	nsubj	O	sentenciação
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	p	p	PROPN	_	Gender=Masc|Number=Sing	1	flat:name	O	_
 4	i	i	PROPN	_	Number=Sing	3	flat:name	O	_
@@ -163056,7 +163056,7 @@
 89-90	nessas	_	_	_	_	_	_	_	_
 89	em	em	ADP	_	_	91	case	O	_
 90	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	91	det	O	_
-91	direções	direção	NOUN	_	Gender=Fem|Number=Plur	87	nmod	O	_
+91	direções	direção	NOUN	_	Gender=Fem|Number=Plur	88	obl	O	_
 92	e	e	CCONJ	_	_	93	cc	O	_
 93	definir	definir	VERB	_	VerbForm=Inf	85	conj	O	_
 94	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	95	det	O	_
@@ -164212,7 +164212,7 @@
 4-5	nas	_	_	_	_	_	_	_	_
 4	em	em	ADP	_	_	6	case	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
-6	linhas	linha	NOUN	_	Gender=Fem|Number=Plur	2	nmod	O	_
+6	linhas	linha	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 7	0231-1328	0231-1328	NUM	_	NumType=Card	6	nummod	O	_
 8	e	e	CCONJ	_	_	9	cc	O	_
 9	0231-1331	0231-1331	NUM	_	NumType=Card	7	conj	O	_
@@ -164517,7 +164517,7 @@
 19-20	ao	_	_	_	_	_	_	_	_
 19	a	a	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	Graben	Graben	PROPN	_	Gender=Masc|Number=Sing	17	nmod	B=LOCAL:GEO	_
+21	Graben	Graben	PROPN	_	Gender=Masc|Number=Sing	18	obl	B=LOCAL:GEO	_
 22	Merluza	Merluza	PROPN	_	Number=Sing	21	flat:name	B=CAMPO	_
 23	e	e	CCONJ	_	_	25	cc	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
@@ -166585,7 +166585,7 @@
 16	presentes	presente	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	em	em	ADP	_	_	19	case	O	_
 18	maior	grande	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
-19	proporção	proporção	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
+19	proporção	proporção	NOUN	_	Gender=Fem|Number=Sing	16	obl	O	_
 20	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	22	cop	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	carbono	carbono	NOUN	_	Gender=Masc|Number=Sing	12	ccomp	O	_
@@ -168514,7 +168514,7 @@
 3	existentes	existente	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
 4	para	para	ADP	_	_	7	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-6	10	10	NUM	_	NumType=Card	7	nummod	O	_
+6	10	10	NUM	_	NumType=Card	7	nummod	O	tokenização
 7	transporte	transporte	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 8-9	deste	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
@@ -170289,7 +170289,7 @@
 34-35	pelo	_	_	_	_	_	_	_	_
 34	por	por	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
-36	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
+36	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	33	obl	O	_
 37	,	,	PUNCT	_	_	42	punct	O	_
 38	caso	caso	SCONJ	_	_	42	mark	O	_
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
@@ -170402,7 +170402,7 @@
 13-14	das	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
-15	forças	força	NOUN	_	Gender=Fem|Number=Plur	11	nmod	O	_
+15	forças	força	NOUN	_	Gender=Fem|Number=Plur	12	obl	O	_
 16	coesivas	coesivo	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
 17	,	,	PUNCT	_	_	20	punct	O	_
 18	entre	entre	ADP	_	_	20	case	O	_
@@ -185337,7 +185337,7 @@
 4-5	da	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
-6	descarga	descarga	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+6	descarga	descarga	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
@@ -193401,7 +193401,7 @@
 8-9	dos	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
-10	reagentes	reagente	NOUN	_	Gender=Masc|Number=Plur	6	nmod	O	_
+10	reagentes	reagente	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
 11	e	e	CCONJ	_	_	13	cc	O	_
 12	ou	ou	CCONJ	_	_	13	cc	O	_
 13	produtos	produto	NOUN	_	Gender=Masc|Number=Plur	10	conj	O	_
@@ -193770,7 +193770,7 @@
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	interior	interior	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
+21	interior	interior	NOUN	_	Gender=Masc|Number=Sing	18	obl	O	_
 22	ou	ou	CCONJ	_	_	23	cc	O	_
 23	suporte	suporte	NOUN	_	Gender=Masc|Number=Sing	21	conj	O	_
 24	.	.	PUNCT	_	_	5	punct	O	_
@@ -198432,7 +198432,7 @@
 4-5	da	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
-6	operação	operação	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+6	operação	operação	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	ser	ser	AUX	_	VerbForm=Inf	9	aux:pass	O	_
 9	divididos	dividir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	7	xcomp	O	_
@@ -199174,7 +199174,7 @@
 15-16	da	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
-17	difração	difração	NOUN	_	Gender=Fem|Number=Sing	13	nmod	O	_
+17	difração	difração	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
 18-19	dos	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
@@ -205116,7 +205116,7 @@
 5-6	nos	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	7	det	O	_
-7	insumos	insumo	NOUN	_	Gender=Masc|Number=Plur	3	nmod	O	_
+7	insumos	insumo	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	produção	produção	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 10	já	já	ADV	_	_	12	advmod	O	_
@@ -213121,11 +213121,11 @@
 16	girassol	girassol	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
 17	e	e	CCONJ	_	_	19	cc	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
-20	gordura	gordura	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
-21	animal	animal	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
+19	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	6	conj	O	_
+20	gordura	gordura	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
+21	animal	animal	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22	e	e	CCONJ	_	_	23	cc	O	_
-23	coco	coco	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
+23	coco	coco	NOUN	_	Gender=Masc|Number=Sing	20	conj	O	_
 24	,	,	PUNCT	_	_	26	punct	O	_
 25	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	26	aux:pass	O	_
 26	utilizado	utilizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	5	conj	O	_
@@ -213133,7 +213133,7 @@
 28	agente	agente	NOUN	_	Gender=Masc|Number=Sing	26	xcomp	O	_
 29	precipitante	precipitante	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
 30	a	a	ADP	_	_	31	case	O	_
-31	barrilha	barrilha	NOUN	_	Gender=Masc|Number=Sing	28	nmod	O	_
+31	barrilha	barrilha	NOUN	_	Gender=Masc|Number=Sing	26	nsubj:pass	O	_
 32	.	.	PUNCT	_	_	5	punct	O	_
 33	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -216816,7 +216816,7 @@
 11	carbetos	carbeto	NOUN	_	Gender=Masc|Number=Plur	7	nmod	O	_
 12	com	com	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	função	função	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
+14	função	função	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 15	ácida	ácido	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	fornecida	fornecer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	14	acl	O	_
 17-18	pelo	_	_	_	_	_	_	_	_
@@ -231726,7 +231726,7 @@
 16	ambientes	ambiente	NOUN	_	Gender=Masc|Number=Plur	9	obl	O	_
 17	carentes	carente	ADJ	_	Gender=Masc|Number=Plur	16	amod	O	_
 18	de	de	ADP	_	_	19	case	O	_
-19	informação	informação	NOUN	_	Gender=Fem|Number=Sing	16	nmod	O	_
+19	informação	informação	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
 20	geológica	geológico	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	direta	direto	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 22	,	,	PUNCT	_	_	9	punct	O	_
@@ -233443,7 +233443,7 @@
 22-23	dos	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	24	det	O	_
-24	esforços	esforço	NOUN	_	Gender=Masc|Number=Plur	20	nmod	O	_
+24	esforços	esforço	NOUN	_	Gender=Masc|Number=Plur	21	obl	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 27	reinantes	reinante	ADJ	_	Gender=Masc|Number=Plur	26	amod	O	_
@@ -233760,7 +233760,7 @@
 12-13	da	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	margem	margem	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
+14	margem	margem	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15	continental	continental	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	sudeste	sudeste	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 17	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	19	nsubj	O	_
@@ -236457,7 +236457,7 @@
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	associação	associação	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
+7	associação	associação	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8-9	destas	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	10	det	O	_
@@ -237188,7 +237188,7 @@
 21-22	do	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
-23	cinturão	cinturão	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
+23	cinturão	cinturão	NOUN	_	Gender=Masc|Number=Sing	20	obl	O	_
 24	móvel	móvel	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 25	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	26	nsubj	O	_
 26	contorna	contornar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	acl:relcl	O	_
@@ -239694,7 +239694,7 @@
 16-17	das	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	18	det	O	_
-18	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=BACIA:TIPO	_
+18	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	15	obl	B=BACIA:TIPO	_
 19	de	de	ADP	_	_	20	case	O	_
 20	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	18	nmod	B=BACIA|B=LOCAL:POL	_
 21	e	e	CCONJ	_	_	22	cc	O	_
@@ -240127,7 +240127,7 @@
 91	altimétricas	altimétrico	ADJ	_	Gender=Fem|Number=Plur	90	amod	O	_
 92	coerentes	coerente	ADJ	_	Gender=Fem|Number=Plur	90	amod	O	_
 93	com	com	ADP	_	_	94	case	O	_
-94	patamares	patamar	NOUN	_	Gender=Masc|Number=Plur	90	nmod	O	_
+94	patamares	patamar	NOUN	_	Gender=Masc|Number=Plur	92	obl	O	_
 95	tectônicos	tectônico	ADJ	_	Gender=Masc|Number=Plur	94	amod	O	_
 96	escalonados	escalonar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	94	acl	O	_
 97	desde	desde	ADP	_	_	99	case	O	_
@@ -244342,7 +244342,7 @@
 29	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	28	amod	O	_
 30	de	de	ADP	_	_	32	case	O	_
 31	múltiplas	múltiplo	ADJ	_	Gender=Fem|Number=Plur	32	amod	O	_
-32	fontes	fonte	NOUN	_	Gender=Fem|Number=Plur	28	nmod	O	_
+32	fontes	fonte	NOUN	_	Gender=Fem|Number=Plur	29	obl	O	_
 33	interferentes	interferente	ADJ	_	Gender=Fem|Number=Plur	32	amod	O	_
 34	.	.	PUNCT	_	_	4	punct	O	_
 35	.	.	PUNCT	_	_	4	punct	O	_
@@ -246022,7 +246022,7 @@
 24	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	21	obl	O	_
 25	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	24	amod	O	_
 26	de	de	ADP	_	_	27	case	O	_
-27	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
+27	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
 28	induzida	induzir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	27	acl	O	_
 29	.	.	PUNCT	_	_	3	punct	O	_
 30	.	.	PUNCT	_	_	3	punct	O	_
@@ -247971,7 +247971,7 @@
 51-52	do	_	_	_	_	_	_	_	_
 51	de	de	ADP	_	_	53	case	O	_
 52	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	53	det	O	_
-53	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	49	nmod	O	_
+53	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	50	obl	O	_
 54	.	.	PUNCT	_	_	42	punct	O	_
 55	.	.	PUNCT	_	_	10	punct	O	_
 
@@ -250386,7 +250386,7 @@
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+10	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 11	somado	somar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	10	acl	O	_
 12	de	de	ADP	_	_	14	case	O	_
 13	dois	dois	NUM	_	NumType=Card	14	nummod	O	_
@@ -255152,7 +255152,7 @@
 22-23	da	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
-24	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
+24	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 25	total	total	ADJ	_	Gender=Fem|Number=Sing	24	amod	O	_
 26	.	.	PUNCT	_	_	13	punct	O	_
 27	.	.	PUNCT	_	_	13	punct	O	_
@@ -256411,7 +256411,7 @@
 25-26	da	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	inversão	inversão	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
+27	inversão	inversão	NOUN	_	Gender=Fem|Number=Sing	24	obl	O	_
 28	compacta	compacto	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
 29	(	(	PUNCT	_	_	33	punct	O	_
 30	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	33	cc	O	_
@@ -257045,7 +257045,7 @@
 3	de	de	ADP	_	_	6	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
 5	última	último	ADJ	_	Gender=Fem|NumType=Ord|Number=Sing	6	amod	O	_
-6	iteração	iteração	NOUN	_	Gender=Fem|Number=Sing	1	nmod	O	_
+6	iteração	iteração	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
 7	K..	K..	PROPN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 
 # text = A representação gráfica é apresentada em duas opções: - escala de cinzas que associa uma intensidade de cinza ao valor de susceptibilidade estimado para cada prisma do modelo de subsuperficie; - escala de cores que corresponde à forma suavizada onde os intervalos de valores de susceptibilidade estimados são representados por áreas coloridas..
@@ -263559,7 +263559,7 @@
 24	proeminente	proeminente	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
 25	,	,	PUNCT	_	_	16	punct	O	_
 26	de	de	ADP	_	_	27	case	O	_
-27	forma	forma	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
+27	forma	forma	NOUN	_	Gender=Fem|Number=Sing	16	obl	O	_
 28	aproximadamente	aproximadamente	ADV	_	_	29	advmod	O	_
 29	perpendicular	perpendicular	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
 30-31	à	_	_	_	_	_	_	_	_
@@ -263611,7 +263611,7 @@
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	aplicação	aplicação	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+7	aplicação	aplicação	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8-9	da	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
@@ -268235,7 +268235,7 @@
 35	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	33	obl	O	_
 36	remanescentes	remanescente	ADJ	_	Gender=Masc|Number=Plur	35	amod	O	_
 37	de	de	ADP	_	_	38	case	O	_
-38	magnetizações	magnetização	NOUN	_	Gender=Fem|Number=Plur	35	nmod	O	_
+38	magnetizações	magnetização	NOUN	_	Gender=Fem|Number=Plur	36	obl	O	_
 39	registrando	registrar	VERB	_	VerbForm=Ger	33	advcl	O	_
 40	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	41	det	O	_
 41	oscilações	oscilação	NOUN	_	Gender=Fem|Number=Plur	39	obj	O	_
@@ -270078,7 +270078,7 @@
 15-16	do	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
-17	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	11	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+17	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	14	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	cristalino	cristalino	ADJ	_	Gender=Masc|Number=Sing	17	amod	B=LITOLOGIA	_
 19	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	21	nsubj	O	_
 20	possivelmente	possivelmente	ADV	_	_	21	advmod	O	_
@@ -270430,7 +270430,7 @@
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
-30	terreno	terreno	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
+30	terreno	terreno	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
 31	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	30	nmod	O	_
 32	Frio	Frio	PROPN	_	Number=Sing	31	flat:name	O	_
 33	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	30	acl	O	_
@@ -270559,7 +270559,7 @@
 21	contrastante	contrastante	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22	com	com	ADP	_	_	24	case	O	_
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
-24	anterior	anterior	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
+24	anterior	anterior	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 25	.	.	PUNCT	_	_	5	punct	O	_
 26	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -272613,7 +272613,7 @@
 6-7	ao	_	_	_	_	_	_	_	_
 6	a	a	ADP	_	_	8	case	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	dique	dique	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
+8	dique	dique	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -274556,12 +274556,12 @@
 21-22	do	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
-23	magmatismo	magmatismo	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
+23	magmatismo	magmatismo	NOUN	_	Gender=Masc|Number=Sing	20	obl	O	_
 24	decorrente	decorrente	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 25-26	das	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	27	det	O	_
-27	reativações	reativação	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
+27	reativações	reativação	NOUN	_	Gender=Fem|Number=Plur	24	obl	O	_
 28	posteriores	posterior	ADJ	_	Gender=Fem|Number=Plur	27	amod	O	_
 29-30	à	_	_	_	_	_	_	_	_
 29	a	a	ADP	_	_	31	case	O	_
@@ -274663,7 +274663,7 @@
 25-26	do	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	contexto	contexto	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
+27	contexto	contexto	NOUN	_	Gender=Masc|Number=Sing	24	obl	O	_
 28	regional	regional	ADJ	_	Gender=Masc|Number=Sing	27	amod	O	_
 29	NE-SW	NE-SW	PROPN	_	Gender=Masc|Number=Sing	27	nmod	O	_
 30	,	,	PUNCT	_	_	31	punct	O	_
@@ -275316,7 +275316,7 @@
 8-9	desta	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	10	det	O	_
-10	forma	forma	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
+10	forma	forma	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
 13	corresponde	corresponder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -275329,12 +275329,12 @@
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 19	Marambaia	Marambaia	PROPN	_	Gender=Fem|Number=Sing	16	nmod	O	_
 20	(	(	PUNCT	_	_	21	punct	O	_
-21	figura	figura	NOUN	_	Gender=Fem|Number=Sing	13	parataxis	O	_
+21	figura	figura	NOUN	_	Gender=Fem|Number=Sing	16	parataxis	O	_
 22	6.12	6.12	NUM	_	NumType=Card	21	nummod	O	_
 23	)	)	PUNCT	_	_	21	punct	O	_
 24	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	26	nsubj:pass	O	_
 25	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	26	aux:pass	O	_
-26	cortado	cortar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	19	acl:relcl	O	_
+26	cortado	cortar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	16	acl:relcl	O	_
 27-28	pelo	_	_	_	_	_	_	_	_
 27	por	por	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -276596,15 +276596,15 @@
 1	Seriam	ser	AUX	_	Mood=Cnd|Number=Plur|Person=3|VerbForm=Fin	2	cop	O	_
 2	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	0	root	O	_
 3	,	,	PUNCT	_	_	4	punct	O	_
-4	portanto	portanto	CCONJ	_	_	7	cc	O	_
+4	portanto	portanto	CCONJ	_	_	2	cc	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	origem	origem	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 8	diferente	diferente	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9-10	daqueles	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	10	case	O	_
-10	aqueles	aquele	DET	_	Gender=Masc|Number=Plur|PronType=Dem	7	nmod	O	_
-11	observados	observar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	7	acl	O	_
+10	aqueles	aquele	DET	_	Gender=Masc|Number=Plur|PronType=Dem	8	obl	O	_
+11	observados	observar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	10	acl	O	_
 12-13	na	_	_	_	_	_	_	_	_
 12	em	em	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
@@ -276615,10 +276615,10 @@
 17	de	de	ADP	_	_	16	flat:name	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
 19	Cabo	Cabo	PROPN	_	Number=Sing	16	flat:name	O	_
-20	,	,	PUNCT	_	_	11	punct	O	_
+20	,	,	PUNCT	_	_	23	punct	O	_
 21	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	23	nsubj	O	_
 22	seriam	ser	AUX	_	Mood=Cnd|Number=Plur|Person=3|VerbForm=Fin	23	cop	O	_
-23	resultantes	resultante	ADJ	_	Gender=Masc|Number=Plur	14	acl:relcl	O	_
+23	resultantes	resultante	ADJ	_	Gender=Masc|Number=Plur	10	acl:relcl	O	_
 24-25	dos	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
@@ -276629,31 +276629,31 @@
 30-31	do	_	_	_	_	_	_	_	_
 30	de	de	ADP	_	_	32	case	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
-32	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	28	nmod	B=GEOCRONOLOGIA	_
+32	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	26	nmod	B=GEOCRONOLOGIA	_
 33	Inferior	Inferior	PROPN	_	Number=Sing	32	flat:name	I=GEOCRONOLOGIA	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	fase	fase	NOUN	_	Gender=Fem|Number=Sing	32	appos	O	_
 36	rifte	rifte	ADJ	_	Gender=Fem|Number=Sing	35	amod	B=LOCAL:GEO	_
 37	)	)	PUNCT	_	_	35	punct	O	_
-38	constituintes	constituinte	ADJ	_	Gender=Masc|Number=Plur	32	amod	O	_
+38	constituintes	constituinte	ADJ	_	Gender=Masc|Number=Plur	26	amod	O	_
 39-40	da	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
 41	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	38	obl	O	_
 42	São	São	PROPN	_	Gender=Fem|Number=Sing	41	nmod	O	_
-43	Paulo-Rio	Paulo-Rio	PROPN	_	Number=Sing	42	flat:name	O	_
+43	Paulo-Rio	Paulo-Rio	PROPN	_	Number=Sing	42	flat:name	O	tokenização
 44	de	de	ADP	_	_	42	flat:name	O	_
 45	Janeiro	Janeiro	PROPN	_	Number=Sing	42	flat:name	O	_
-46	responsável	responsável	ADJ	_	Gender=Masc|Number=Sing	42	amod	O	_
+46	responsável	responsável	ADJ	_	Gender=Masc|Number=Sing	41	amod	O	_
 47	,	,	PUNCT	_	_	50	punct	O	_
 48	em	em	ADP	_	_	50	case	O	_
 49	grande	grande	ADJ	_	Gender=Fem|Number=Sing	50	amod	O	_
 50	parte	parte	NOUN	_	Gender=Fem|Number=Sing	46	obl	O	_
-51	,	,	PUNCT	_	_	56	punct	O	_
+51	,	,	PUNCT	_	_	50	punct	O	_
 52-53	pela	_	_	_	_	_	_	_	_
 52	por	por	ADP	_	_	54	case	O	_
 53	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	54	det	O	_
-54	assinatura	assinatura	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
+54	assinatura	assinatura	NOUN	_	Gender=Fem|Number=Sing	46	nmod	O	_
 55	magnética	magnético	ADJ	_	Gender=Fem|Number=Sing	54	amod	O	_
 56	observada	observar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	54	acl	O	_
 57-58	no	_	_	_	_	_	_	_	_
@@ -277386,7 +277386,7 @@
 33-34	dos	_	_	_	_	_	_	_	_
 33	de	de	ADP	_	_	35	case	O	_
 34	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	35	det	O	_
-35	flancos	flanco	NOUN	_	Gender=Masc|Number=Plur	30	nmod	O	_
+35	flancos	flanco	NOUN	_	Gender=Masc|Number=Plur	32	obl	O	_
 36	sul	sul	ADJ	_	Gender=Masc|Number=Sing	35	amod	O	_
 37	e	e	CCONJ	_	_	38	cc	O	_
 38	norte	norte	ADJ	_	Gender=Masc|Number=Sing	36	conj	O	_
@@ -280547,7 +280547,7 @@
 20-21	dos	_	_	_	_	_	_	_	_
 20	de	de	ADP	_	_	22	case	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	22	det	O	_
-22	processos	processo	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
+22	processos	processo	NOUN	_	Gender=Masc|Number=Plur	19	obl	O	_
 23	magmáticos	magmático	ADJ	_	Gender=Masc|Number=Plur	22	amod	O	_
 24	constituem	constituir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	ccomp	O	_
 25	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	26	det	O	_
@@ -281217,7 +281217,7 @@
 25	para	para	ADP	_	_	28	case	O	_
 26	ambos	ambos	DET	_	Gender=Masc|Number=Plur|PronType=Tot	28	det	O	_
 27	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	28	det	O	_
-28	casos	caso	NOUN	_	Gender=Masc|Number=Plur	22	nmod	O	_
+28	casos	caso	NOUN	_	Gender=Masc|Number=Plur	24	obl	O	_
 29	.	.	PUNCT	_	_	15	punct	O	_
 30	.	.	PUNCT	_	_	15	punct	O	_
 
@@ -281610,7 +281610,7 @@
 27	condizentes	condizente	ADJ	_	Gender=Fem|Number=Plur	25	amod	O	_
 28	com	com	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
-30	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
+30	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
 31	geológico	geológico	ADJ	_	Gender=Masc|Number=Sing	30	amod	O	_
 32	investigado	investigar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	30	acl	O	_
 33	.	.	PUNCT	_	_	6	punct	O	_
@@ -281669,7 +281669,7 @@
 17	ambientes	ambiente	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
 18	carentes	carente	ADJ	_	Gender=Masc|Number=Plur	17	amod	O	_
 19	de	de	ADP	_	_	20	case	O	_
-20	informação	informação	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+20	informação	informação	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 21	a	a	ADP	_	_	22	case	O	_
 22	priori	priori	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 23	sobre	sobre	ADP	_	_	25	case	O	_
@@ -282371,7 +282371,7 @@
 28	resultantes	resultante	ADJ	_	Gender=Fem|Number=Plur	27	amod	O	_
 29	de	de	ADP	_	_	31	case	O	_
 30	múltiplas	múltiplo	ADJ	_	Gender=Fem|Number=Plur	31	amod	O	_
-31	fontes	fonte	NOUN	_	Gender=Fem|Number=Plur	27	nmod	O	_
+31	fontes	fonte	NOUN	_	Gender=Fem|Number=Plur	28	obl	O	_
 32	interferentes	interferente	ADJ	_	Gender=Fem|Number=Plur	31	amod	O	_
 33	.	.	PUNCT	_	_	11	punct	O	_
 34	.	.	PUNCT	_	_	11	punct	O	_
@@ -283298,7 +283298,7 @@
 57-58	nos	_	_	_	_	_	_	_	_
 57	em	em	ADP	_	_	59	case	O	_
 58	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	59	det	O	_
-59	dados	dado	NOUN	_	Gender=Masc|Number=Plur	55	nmod	O	_
+59	dados	dado	NOUN	_	Gender=Masc|Number=Plur	56	obl	O	_
 60	.	.	PUNCT	_	_	19	punct	O	_
 61	.	.	PUNCT	_	_	19	punct	O	_
 
@@ -283598,7 +283598,7 @@
 7	geográfica	geográfico	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
 8	conveniente	conveniente	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
 9	para	para	ADP	_	_	10	case	O	_
-10	polarização	polarização	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
+10	polarização	polarização	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 11	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	12	nsubj	O	_
 12	resultou	resultar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	10	acl:relcl	O	_
 13-14	na	_	_	_	_	_	_	_	_
@@ -284355,7 +284355,7 @@
 19	dificilmente	dificilmente	ADV	_	_	20	advmod	O	_
 20	explicáveis	explicável	ADJ	_	Gender=Fem|Number=Plur	18	amod	O	_
 21	por	por	ADP	_	_	22	case	O	_
-22	indução	indução	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
+22	indução	indução	NOUN	_	Gender=Fem|Number=Sing	20	obl	O	_
 23	provocada	provocar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	22	acl	O	_
 24	unicamente	unicamente	ADV	_	_	23	advmod	O	_
 25-26	pelo	_	_	_	_	_	_	_	_
@@ -287256,7 +287256,7 @@
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
 23	não	não	ADV	_	Polarity=Neg	24	advmod	O	_
-24	satisfaçam	satisfaçor	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	advcl	O	_
+24	satisfaçam	satisfazer	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	33	advcl	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 26	valores	valor	NOUN	_	Gender=Masc|Number=Plur	24	obj	O	_
 27	de	de	ADP	_	_	28	case	O	_
@@ -287266,16 +287266,16 @@
 31	adequados	adequado	ADJ	_	Gender=Masc|Number=Plur	30	amod	O	_
 32	,	,	PUNCT	_	_	33	punct	O	_
 33-34	dilui-se	_	_	_	_	_	_	_	_
-33	dilui	diluir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
+33	dilui	diluir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Pass	5	conj	O	_
 34	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	33	nsubj	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
-36	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	33	obj	O	_
+36	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	33	nsubj:pass	O	_
 37	(	(	PUNCT	_	_	42	punct	O	_
-38	quando	quando	ADV	_	_	42	advmod	O	_
+38	quando	quando	SCONJ	_	_	42	mark	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
 40	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	42	nsubj	O	_
 41	estiver	estar	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	42	cop	O	_
-42	acima	acima	ADV	_	_	48	advmod	O	_
+42	acima	acima	ADV	_	_	33	advcl	O	_
 43-44	da	_	_	_	_	_	_	_	_
 43	de	de	ADP	_	_	45	case	O	_
 44	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	45	det	O	_
@@ -287283,35 +287283,35 @@
 46	)	)	PUNCT	_	_	42	punct	O	_
 47	ou	ou	CCONJ	_	_	48	cc	O	_
 48-49	adiciona-se	_	_	_	_	_	_	_	_
-48	adiciona	adicionar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	33	conj	O	_
+48	adiciona	adicionar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Pass	33	conj	O	_
 49	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	48	expl	O	_
 50	,	,	PUNCT	_	_	48	punct	O	_
 51-52	na	_	_	_	_	_	_	_	_
 51	em	em	ADP	_	_	53	case	O	_
 52	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	53	det	O	_
 53	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	48	obl	O	_
-54	,	,	PUNCT	_	_	55	punct	O	_
-55	argilas	argila	NOUN	_	Gender=Fem|Number=Plur	53	conj	B=LITOLOGIA	_
+54	,	,	PUNCT	_	_	53	punct	O	_
+55	argilas	argila	NOUN	_	Gender=Fem|Number=Plur	48	nsubj:pass	B=LITOLOGIA	_
 56	com	com	ADP	_	_	57	case	O	_
-57	características	característica	NOUN	_	Gender=Fem|Number=Plur	48	obl	O	_
+57	características	característica	NOUN	_	Gender=Fem|Number=Plur	55	nmod	O	_
 58	viscosificantes	viscosificante	ADJ	_	Gender=Fem|Number=Plur	57	amod	B=QUÍMICA	_
-59	(	(	PUNCT	_	_	67	punct	O	_
-60	quando	quando	ADV	_	_	64	advmod	O	_
+59	(	(	PUNCT	_	_	64	punct	O	_
+60	quando	quando	SCONJ	_	_	64	mark	O	_
 61	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	62	det	O	_
 62	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	64	nsubj	O	_
-63	estiver	estar	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	67	cop	O	_
-64	abaixo	abaixo	ADV	_	_	67	advmod	O	_
+63	estiver	estar	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	64	cop	O	_
+64	abaixo	abaixo	ADJ	_	_	48	advcl	O	_
 65-66	da	_	_	_	_	_	_	_	_
 65	de	de	ADP	_	_	67	case	O	_
 66	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	67	det	O	_
-67	desejada	desejar	NOUN	_	Gender=Fem|Number=Sing	57	nmod	O	_
-68	)	)	PUNCT	_	_	67	punct	O	_
+67	desejada	desejar	NOUN	_	Gender=Fem|Number=Sing	64	obl	O	_
+68	)	)	PUNCT	_	_	64	punct	O	_
 69	,	,	PUNCT	_	_	71	punct	O	_
 70	entre	entre	ADP	_	_	71	case	O	_
 71	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	74	nmod	O	_
 72	,	,	PUNCT	_	_	71	punct	O	_
 73	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	74	det	O	_
-74	bentonita	bentonita	NOUN	_	Gender=Fem|Number=Sing	57	appos	B=LITOLOGIA	_
+74	bentonita	bentonita	NOUN	_	Gender=Fem|Number=Sing	55	nmod:appos	B=LITOLOGIA	_
 75	.	.	PUNCT	_	_	5	punct	O	_
 76	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -287944,7 +287944,7 @@
 16	constante	constante	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 17	de	de	ADP	_	_	19	case	O	_
 18	20	20	NUM	_	NumType=Card	19	nummod	O	_
-19	psi	psi	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
+19	psi	psi	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 20	(	(	PUNCT	_	_	23	punct	O	_
 21	~	~	SYM	_	_	23	nummod	O	_
 22	138	138	NUM	_	NumType=Card	23	nummod	O	_
@@ -289531,7 +289531,7 @@
 24-25	no	_	_	_	_	_	_	_	_
 24	em	em	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
-26	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
+26	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	23	obl	O	_
 27-28	no	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -298925,7 +298925,7 @@
 5	e	e	CCONJ	_	_	6	cc	O	_
 6	injeção	injeção	NOUN	_	Gender=Fem|Number=Sing	4	conj	O	_
 7	em	em	ADP	_	_	8	case	O	_
-8	poços	poço	NOUN	_	Gender=Masc|Number=Plur	1	nmod	B=POÇO:TIPO	_
+8	poços	poço	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=POÇO:TIPO	_
 9	de	de	ADP	_	_	10	case	I=POÇO:TIPO	_
 10	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	8	nmod	I=POÇO:TIPO|B=FLUIDO	_
 11	e	e	CCONJ	_	_	12	cc	O	_
@@ -299978,7 +299978,7 @@
 12	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 13	molhável	molhável	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	a	a	ADP	_	_	15	case	O	_
-15	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=FLUIDO	_
+15	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	13	obl	B=FLUIDO	_
 16	tende	tender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 17	a	a	SCONJ	_	_	18	mark	O	_
 18	danificar	danificar	VERB	_	VerbForm=Inf	16	xcomp	O	_
@@ -305746,7 +305746,7 @@
 14-15	da	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	amostra	amostra	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
+16	amostra	amostra	NOUN	_	Gender=Fem|Number=Sing	13	obl	O	_
 17	e	e	CCONJ	_	_	39	cc	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
 19	passado	passar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	39	advcl	O	_
@@ -310350,7 +310350,7 @@
 6-7	na	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	redução	redução	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
+8	redução	redução	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -310759,7 +310759,7 @@
 23	de	de	ADP	_	_	26	case	O	_
 24	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 25	respectivos	respectivo	ADJ	_	Gender=Masc|Number=Plur	26	amod	O	_
-26	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	20	nmod	O	_
+26	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	22	obl	O	_
 27	sob	sob	ADP	_	_	29	case	O	_
 28	baixas	baixo	ADJ	_	Gender=Fem|Number=Plur	29	amod	O	_
 29	taxas	taxa	NOUN	_	Gender=Fem|Number=Plur	20	nmod	O	_
@@ -313995,7 +313995,7 @@
 11-12	deste	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	13	det	O	_
-13	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
+13	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	,	,	PUNCT	_	_	18	punct	O	_
 15	bem	bem	ADP	_	_	18	case	O	_
 16	como	como	ADP	_	_	15	fixed	O	_
@@ -315287,7 +315287,7 @@
 20-21	ao	_	_	_	_	_	_	_	_
 20	a	a	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	18	nmod	B=FLUIDO	_
+22	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	19	obl	B=FLUIDO	_
 23	tende	tender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 24	a	a	SCONJ	_	_	25	mark	O	_
 25	danificar	danificar	VERB	_	VerbForm=Inf	23	xcomp	O	_
@@ -317493,7 +317493,7 @@
 5	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	circulante	circulante	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	ar	ar	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
+8	ar	ar	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 9	,	,	PUNCT	_	_	10	punct	O	_
 10	gás	gás	NOUN	_	Gender=Masc|Number=Sing	8	conj	B=FLUIDO	_
 11	natural	natural	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
@@ -320395,7 +320395,7 @@
 31-32	do	_	_	_	_	_	_	_	_
 31	de	de	ADP	_	_	33	case	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
-33	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	27	nmod	O	_
+33	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	30	obl	O	_
 34	.	.	PUNCT	_	_	4	punct	O	_
 35	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -321653,7 +321653,7 @@
 32-33	nos	_	_	_	_	_	_	_	_
 32	em	em	ADP	_	_	34	case	O	_
 33	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	34	det	O	_
-34	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	30	nmod	O	_
+34	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	31	obl	O	_
 35	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	37	nsubj	O	_
 36	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	37	expl	O	_
 37	encontram	encontrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	34	acl:relcl	O	_
@@ -329055,7 +329055,7 @@
 12	relevantes	relevante	ADJ	_	Gender=Fem|Number=Plur	11	amod	O	_
 13	para	para	ADP	_	_	15	case	O	_
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
-15	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	11	nmod	B=FLUIDO	_
+15	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	12	obl	B=FLUIDO	_
 16	de	de	ADP	_	_	17	case	I=FLUIDO	_
 17	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	15	nmod	B=LITOLOGIA|I=FLUIDO	_
 18	.	.	PUNCT	_	_	6	punct	O	_
@@ -332651,7 +332651,7 @@
 3	presente	presente	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	sobre	sobre	ADP	_	_	6	case	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
-6	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	2	nmod	O	_
+6	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 7-8	dos	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
@@ -332880,12 +332880,12 @@
 41-42	nos	_	_	_	_	_	_	_	_
 41	em	em	ADP	_	_	43	case	O	_
 42	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	43	det	O	_
-43	hidrocarbonetos	hidrocarboneto	NOUN	_	Gender=Masc|Number=Plur	39	nmod	O	_
+43	hidrocarbonetos	hidrocarboneto	NOUN	_	Gender=Masc|Number=Plur	40	obl	O	_
 44	,	,	PUNCT	_	_	37	punct	O	_
 45	onde	onde	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	48	obl	O	_
 46	algumas	algum	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	48	nsubj	O	_
 47	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	48	cop	O	_
-48	impurezas	impureza	NOUN	_	Gender=Fem|Number=Plur	43	acl:relcl	O	_
+48	impurezas	impureza	NOUN	_	Gender=Fem|Number=Plur	39	acl:relcl	O	_
 49	e	e	CCONJ	_	_	50	cc	O	_
 50	matérias	matéria	NOUN	_	Gender=Fem|Number=Plur	48	conj	O	_
 51	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	52	nsubj	O	_
@@ -333997,7 +333997,7 @@
 32-33	no	_	_	_	_	_	_	_	_
 32	em	em	ADP	_	_	34	case	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
-34	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	30	nmod	B=FLUIDO	_
+34	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	31	obl	B=FLUIDO	_
 35	(	(	PUNCT	_	_	36	punct	O	_
 36	THOMAS	THOMAS	PROPN	_	Gender=Masc|Number=Sing	11	parataxis	O	_
 37	,	,	PUNCT	_	_	36	flat:name	O	_
@@ -334405,7 +334405,7 @@
 3	atuantes	atuante	ADJ	_	Gender=Masc|Number=Plur	2	amod	O	_
 4	sobre	sobre	ADP	_	_	6	case	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
-6	moléculas	molécula	NOUN	_	Gender=Fem|Number=Plur	2	nmod	O	_
+6	moléculas	molécula	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 7-8	nesta	_	_	_	_	_	_	_	_
 7	em	em	ADP	_	_	9	case	O	_
 8	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	9	det	O	_
@@ -335326,7 +335326,7 @@
 11-12	ao	_	_	_	_	_	_	_	_
 11	a	a	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	município	município	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
+13	município	município	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	Icapuí	Icapuí	PROPN	_	Gender=Unsp|Number=Sing	13	nmod	O	_
 16	.	.	PUNCT	_	_	5	punct	O	_
@@ -336062,7 +336062,7 @@
 8-9	da	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	deriva	deriva	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
+10	deriva	deriva	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 11	litorânea	litorâneo	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	sudeste	sudeste	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
@@ -336476,7 +336476,7 @@
 18	provenientes	proveniente	ADJ	_	Gender=Masc|Number=Plur	17	amod	O	_
 19	de	de	ADP	_	_	21	case	O	_
 20	vários	vário	DET	_	Gender=Masc|Number=Plur|PronType=Ind	21	det	O	_
-21	ambientes	ambiente	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
+21	ambientes	ambiente	NOUN	_	Gender=Masc|Number=Plur	18	obl	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
 23	ocupam	ocupar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	conj	O	_
 24	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	27	det	O	_
@@ -336750,7 +336750,7 @@
 31-32	nos	_	_	_	_	_	_	_	_
 31	em	em	ADP	_	_	33	case	O	_
 32	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	33	det	O	_
-33	estados	estado	NOUN	_	Gender=Masc|Number=Plur	25	nmod	O	_
+33	estados	estado	NOUN	_	Gender=Masc|Number=Plur	30	nmod	O	_
 34	unidos	unido	ADJ	_	Gender=Masc|Number=Plur	33	amod	O	_
 35	,	,	PUNCT	_	_	11	punct	O	_
 36	até	até	ADP	_	_	37	case	O	_
@@ -337538,7 +337538,7 @@
 59-60	nesse	_	_	_	_	_	_	_	_
 59	em	em	ADP	_	_	61	case	O	_
 60	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	61	det	O	_
-61	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	57	nmod	O	_
+61	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	58	obl	O	_
 62	(	(	PUNCT	_	_	63	punct	O	_
 63	POSSOBOM	POSSOBOM	PROPN	_	Gender=Masc|Number=Sing	6	parataxis	O	_
 64	,	,	PUNCT	_	_	63	flat:name	O	_
@@ -337690,7 +337690,7 @@
 62-63	ao	_	_	_	_	_	_	_	_
 62	a	a	ADP	_	_	64	case	O	_
 63	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	64	det	O	_
-64	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	60	nmod	B=FLUIDO	_
+64	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	61	obl	B=FLUIDO	_
 65	(	(	PUNCT	_	_	66	punct	O	_
 66	BRASIL	Brasil	PROPN	_	Gender=Masc|Number=Sing	12	parataxis	O	_
 67	,	,	PUNCT	_	_	66	flat:name	O	_
@@ -339041,7 +339041,7 @@
 65-66	no	_	_	_	_	_	_	_	_
 65	em	em	ADP	_	_	67	case	O	_
 66	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	67	det	O	_
-67	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	63	nmod	O	_
+67	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	64	obl	O	_
 68	,	,	PUNCT	_	_	70	punct	O	_
 69	quanto	quanto	ADV	_	_	70	advmod	O	_
 70	maior	grande	ADJ	_	Gender=Masc|Number=Sing	81	amod	O	_
@@ -345661,7 +345661,7 @@
 4-5	no	_	_	_	_	_	_	_	_
 4	em	em	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	poro	poro	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	poro	poro	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7-8	da	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -345700,7 +345700,7 @@
 16	em	em	ADP	_	_	19	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 18	própria	próprio	DET	_	Gender=Fem|Number=Sing|PronType=Emp	19	det	O	_
-19	formação	formação	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
+19	formação	formação	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 20	.	.	PUNCT	_	_	7	punct	O	_
 21	.	.	PUNCT	_	_	7	punct	O	_
 
@@ -347133,7 +347133,7 @@
 16-17	dos	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
-18	separadores	separador	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
+18	separadores	separador	NOUN	_	Gender=Masc|Number=Plur	15	obl	O	_
 19	e	e	CCONJ	_	_	20	cc	O	_
 20	tratadores	tratador	NOUN	_	Gender=Masc|Number=Plur	18	conj	O	_
 21	de	de	ADP	_	_	22	case	O	_
